paddle/fluid/pybind/imperative.cc

```diff
@@ -824,6 +851,18 @@ void BindImperative(py::module *m_ptr) {
               return std::vector<int>();
             }
           })
+      .def_property_readonly(
+          "place",
+          [](imperative::VarBase &self) {
+            if (self.Var().IsType<framework::LoDTensor>()) {
+              return self.Var().Get<framework::LoDTensor>().place();
+            } else if (self.Var().IsType<framework::SelectedRows>()) {
+              return self.Var().Get<framework::SelectedRows>().value().place();
+            } else {
+              return paddle::platform::Place();
```

Comment:
if we not support other type, throw error maybe better, we can't give user a error place.

Reply:
Done